search.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/tidwall/gjson"
+)
+
+// DefaultResponseBodyTitle defines the default header for the body view
+const DefaultResponseBodyTitle = "Response body (F9)"
+
+// Search performs a search on the request
+// returns title, display, error
+func Search(request *Request, searchString string) (string, []byte, error) {
+	isBinary := (strings.Index(request.ContentType, "text") == -1 &&
+		strings.Index(request.ContentType, "application") == -1)
```

Comment:
Oops, this is my bad. When I pull out the code, I didn't change the logic from the previous part.